biomage/stage/stage.py

```diff
@@ -129,6 +130,7 @@ def get_all_experiments(source_table="experiments-staging"):
         )
         experiment_ids = [*experiment_ids, *response.get("Items")]
 
+    experiment_ids.sort(key=itemgetter("experimentId"))
```

Comment:
you can also use lambdas which don't require any import like `experiment_ids.sort(key=lambda x: x["experimentId"])

Reply:
I tend to not use lambdas, but I agree that here it makes it easier to read. Thanks!